tasks/shell_crmsh/create-and-push-cib.yml

```diff
@@ -229,6 +238,38 @@
         index_var: constraint_index
         loop_var: constraint
 
+    ## Cluster properties
+    - name: Configure cluster properties
+      ansible.builtin.include_tasks: crm-cluster-properties.yml
+      vars:
+        properties_set: "{{ ha_cluster_cluster_properties[0] }}"
+      when: ha_cluster_cluster_properties[0].attrs | d([])
+
+    # Verify CIB to ensure that there are no errors before applying.
+    - name: Verify shadow CIB
+      ansible.builtin.command:
+        cmd: >-
+          crm_verify -V -x
+          /var/lib/pacemaker/cib/shadow.{{ __ha_cluster_crm_shadow }}
```

Comment:
same here re: use of `quote`

Reply:
this is internal variable, already used for shadow CIB commands, so quote is not needed.